widgets/accordion/js/accordion.js

```diff
@@ -108,7 +108,8 @@ jQuery( function ( $ ) {
 					for ( var i = 0; i < panels.length; i++ ) {
 						var panel = panels[ i ];
 						var anchor = $( panel ).data( 'anchor' );
-						if ( anchor && window.location.hash.indexOf( anchor ) > -1 ) {
+						var windowHash = window.location.hash.substring(1).split( ',' ); 
```

Comment:
The term 'hash' is quite overloaded, so rename this to something a bit more specific. Perhaps just `anchors`?

Reply:
Anchors seems a tad confusing due to `anchor` (which is a list of anchors) being present. Maybe `windowAnchors`?

Reply:
`anchor` isn't a list, it's just one anchor name (a string). So `anchors` makes perfect sense for the array which contains many anchor names (an array of strings).

Reply:
Ah, my b. Didn't pick up on that.